.changeset/pink-carpets-return.md

```diff
@@ -0,0 +1,27 @@
+---
+'sku': patch
+---
+
+Replace `sku/config/jest` entrypoint with `sku/jest-preset`
+
+**BREAKING CHANGE**:
```

Comment:
Maybe scope this message to the API which makes sense why its not considered a formal break:

```suggestion
**BREAKING CHANGE FOR `dangerouslySetJestConfig`**:
```

Thoughts?